Rakefile

```diff
@@ -50,13 +43,13 @@ task 'test:spec' => 'test:spec:run'
 task 'test:acceptance' => 'test:acceptance:quick'
 
 # global defaults
-task :test => 'test:spec'
-task :default => :test
+task test: 'test:spec'
+task default: :test
 
 begin
   require 'rubygems'
   require 'github_changelog_generator/task'
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
```

Comment:
add a comment otherwise rubocop maybe nagging about empty rescue

Reply:
odd that rubocop didn't already complain. I will have a look.